TYPO3.Neos/Classes/TYPO3/Neos/Controller/Module/Administration/SitesController.php

```diff
@@ -345,9 +345,15 @@ public function createDomainAction(Domain $domain)
      */
     public function deleteDomainAction(Domain $domain)
     {
+        $site = $domain->getSite();
+        if ($site->getPrimaryDomain() === $domain) {
+            $domain->setActive(false);
```

Comment:
true, but that happens automatically.. actually it's better not to do that and only store a primary when a site is edited.. see https://github.com/neos/neos-development-collection/blob/master/TYPO3.Neos/Classes/TYPO3/Neos/Domain/Model/Site.php#L305

so I'd suggest to remove line 350 and adjust 351 to `null` instead of `$site->getFirstActiveDomain()`